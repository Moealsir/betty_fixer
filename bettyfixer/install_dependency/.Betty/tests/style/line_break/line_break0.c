int func0(char very_very_very_very_very_long_variable_name_just_to_test_length)
{
																																																																					return (1);
}
