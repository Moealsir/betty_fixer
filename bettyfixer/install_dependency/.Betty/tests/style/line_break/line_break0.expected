tests/style/line_break/line_break0.c:3: WARNING: line over 80 characters (81)
total: 0 errors, 1 warnings, 4 lines checked
